sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -2649,3 +2652,84 @@ case class Sentences(
     copy(str = newFirst, language = newSecond, country = newThird)
 
 }
+
+case class ToPrettyString(expr: Expression, timeZoneId: Option[String] = None)
```

Comment:
`sql/catalyst` is a good place for this expr? How about `sql/hive` instead?

Reply:
> `sql/catalyst` is a good place for this expr? How about `sql/hive` instead?

Sound reasonable , WDYT @cloud-fan